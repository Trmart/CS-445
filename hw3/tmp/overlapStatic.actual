.   Child: 0  Parm: p of type int [line: 4]
.   .   Child: 0  Var: a of static type int [line: 6]
.   .   .   .   Child: 0  Var: b of static type int [line: 15]
.   .   .   Child: 0  Var: b of static type int [line: 8]
.   .   .   Child: 0  Var: c of static type int [line: 11]
.   .   .   .   Child: 0  Var: c of static type int [line: 18]
.   .   .   Child: 1  Compound [line: 14]
.   Child: 1  Compound [line: 5]
.   .   Child: 1  Compound [line: 7]
ERROR(15): Symbol 'b' is already declared at line 8.
ERROR(18): Symbol 'c' is already declared at line 11.
ERROR(SymbolTable): You cannot leave global scope.  Number of scopes: 1.
ERROR(SymbolTable): You cannot leave global scope.  Number of scopes: 1.
ERROR(SymbolTable): You cannot leave global scope.  Number of scopes: 1.
ERROR(SymbolTable): You cannot leave global scope.  Number of scopes: 1.
ERROR(SymbolTable): You cannot leave global scope.  Number of scopes: 1.
ERROR(SymbolTable): You cannot leave global scope.  Number of scopes: 1.
Number of errors: 2
Number of warnings: 23
.   .   Sibling: 1  Compound [line: 10]
.   .   .   Sibling: 1  Compound [line: 17]
Sibling: 1  Func: main returns type void [line: 4]
.   .   Sibling: 2  Compound [line: 13]
Sibling: 2  Var: h of type int [line: 23]
Var: g of type int [line: 2]
WARNING(11): The variable 'c' seems not to be used.
WARNING(11): The variable 'c' seems not to be used.
WARNING(11): The variable 'c' seems not to be used.
WARNING(11): The variable 'c' seems not to be used.
WARNING(11): The variable 'c' seems not to be used.
WARNING(2): The variable 'g' seems not to be used.
WARNING(2): The variable 'g' seems not to be used.
WARNING(2): The variable 'g' seems not to be used.
WARNING(2): The variable 'g' seems not to be used.
WARNING(2): The variable 'g' seems not to be used.
WARNING(2): The variable 'g' seems not to be used.
WARNING(6): The variable 'a' seems not to be used.
WARNING(6): The variable 'a' seems not to be used.
WARNING(6): The variable 'a' seems not to be used.
WARNING(6): The variable 'a' seems not to be used.
WARNING(6): The variable 'a' seems not to be used.
WARNING(6): The variable 'a' seems not to be used.
WARNING(8): The variable 'b' seems not to be used.
WARNING(8): The variable 'b' seems not to be used.
WARNING(8): The variable 'b' seems not to be used.
WARNING(8): The variable 'b' seems not to be used.
WARNING(8): The variable 'b' seems not to be used.
WARNING(8): The variable 'b' seems not to be used.
