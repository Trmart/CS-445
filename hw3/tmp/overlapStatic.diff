1,2c1,10
< ERROR(LINKER): A function named 'main()' must be defined.
< Number of errors: 1
---
> .   Child: 0  Parm: p of type int [line: 4]
> .   .   Child: 0  Var: a of static type int [line: 6]
> .   .   .   .   Child: 0  Var: b of static type int [line: 15]
> .   .   .   Child: 0  Var: b of static type int [line: 8]
> .   .   .   Child: 0  Var: c of static type int [line: 11]
> .   .   .   .   Child: 0  Var: c of static type int [line: 18]
> .   .   .   Child: 1  Compound [line: 14]
> .   Child: 1  Compound [line: 5]
> .   .   Child: 1  Compound [line: 7]
> Number of errors: 0
3a12,17
> .   .   Sibling: 1  Compound [line: 10]
> .   .   .   Sibling: 1  Compound [line: 17]
> Sibling: 1  Func: main returns type void [line: 4]
> .   .   Sibling: 2  Compound [line: 13]
> Sibling: 2  Var: h of type int [line: 23]
> Var: g of type int [line: 2]
7c21
< WARNING(4): The variable 'p' seems not to be used.
---
> WARNING(2): The variable 'g' seems not to be used.
