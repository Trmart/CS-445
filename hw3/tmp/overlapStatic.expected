ERROR(LINKER): A function named 'main()' must be defined.
Number of errors: 1
Number of warnings: 6
WARNING(11): The variable 'c' seems not to be used.
WARNING(15): The variable 'b' seems not to be used.
WARNING(18): The variable 'c' seems not to be used.
WARNING(4): The variable 'p' seems not to be used.
WARNING(6): The variable 'a' seems not to be used.
WARNING(8): The variable 'b' seems not to be used.
